### ISSUE TITLE
vite 5 / rollup 4 crashes due to missing report from node:process

### ISSUE THREAD

**Issue by sartak** [oven-sh/bun#7164]
### What version of Bun is running?

1.0.12+85c997513

### What platform is your computer?

Linux 5.4.0-144-generic x86_64 unknown (inside a Docker container)

### What steps can reproduce the bug?

```
λ curl -fsSL https://bun.sh/install | bash
…
```

```
λ bunx rollup
1 | const { existsSync } = require('node:fs');
2 | const { join } = require('node:path');
3 | const { platform, arch, report } = require('node:process');
4 |
5 | const isMusl = () => !report.getReport().header.glibcVersionRuntime;
                                          ^
TypeError: undefined is not an object (evaluating 'report.getReport')
      at isMusl (/tmp/rollup@latest--bunx/node_modules/rollup/dist/native.js:5:39)
      at /tmp/rollup@latest--bunx/node_modules/rollup/dist/native.js:47:37
      at globalThis (/tmp/rollup@latest--bunx/node_modules/rollup/dist/native.js:55:33)
      at overridableRequire (:1:20)
      at /tmp/rollup@latest--bunx/node_modules/rollup/dist/shared/parseAst.js:12:6
      at globalThis (/tmp/rollup@latest--bunx/node_modules/rollup/dist/shared/parseAst.js:2356:26)
      at overridableRequire (:1:20)
      at /tmp/rollup@latest--bunx/node_modules/rollup/dist/shared/rollup.js:12:6
```

### What is the expected behavior?

`bunx rollup` runs to completion.

### What do you see instead?

Rollup crashes on `node:process.report.getReport()` because `report` is undefined.

### Additional information

This blocks migration of my apps to Vite 5.

It's currently documented that bun's `node:process` has not yet implemented `report` - https://bun.sh/docs/runtime/nodejs-apis#process

**Comment by sartak** [comment]
https://github.com/oven-sh/bun/pull/3326 looks related (especially since its stated purpose is for inspecting libc), but might be insufficient - its `getReport()` appears to return a string, so `getReport().header.glibcVersionRuntime` would presumably still crash.

**Comment by Electroid** [comment]
Fixed by #7171
